pkg/provisioner/ironic/ironic.go

```diff
@@ -55,6 +55,10 @@ const (
 	powerOff     = "power off"
 	softPowerOff = "soft power off"
 	powerNone    = "None"
+
+	// Maximum number of retries an operation returning error
+	// would be retried.
+	maxErrorRetryCount = 3
```

Comment:
The name doesn't seem correct, as it is a threshold currently adopted for limiting the number of retries just for powering off the host

Reply:
True but it could be used for other retries too. I'll try to come up with a better name.

Reply:
Renamed it to maxPowerOffRetryCount. Hopefully, that is a better name.